Sources/BraveVPN/BraveVPN.swift

```diff
@@ -13,6 +13,7 @@ import os.log
 
 /// A static class to handle all things related to the Brave VPN service.
 public class BraveVPN {
+  public static let linkReceiptEnabled = false
```

Comment:
This flag will be removed once all the cross platform vpn functionality is ready